examples/update/canisters/azle/index.ts

```diff
@@ -0,0 +1,36 @@
+import { ic, nat64, Opt, Query, Update } from 'azle';
+
+//#region Performance
+type PerfResult = {
+    wasm_body_only: nat64;
+    wasm_including_prelude: nat64;
+};
+
+let perf_result: Opt<PerfResult> = null;
+
+export function get_perf_result(): Query<Opt<PerfResult>> {
+    return perf_result;
+}
+
+function record_performance(start: nat64, end: nat64): void {
+    perf_result = {
+        wasm_body_only: end - start,
+        wasm_including_prelude: ic.performance_counter(0)
+    };
+}
+//#endregion
+
+let currentMessage: string = '';
+
+export function query(): Query<string> {
```

Comment:
The Azle and Motoko names for this function should be the same, perhaps something like `get_current_message` would be better than `query1`?